python/pyspark/ml/torch/deepspeed/deepspeed_distributer.py

```diff
@@ -0,0 +1,167 @@
+import json
+import os
+import subprocess
+import tempfile
+from typing import (
+    Union,
+    Callable,
+    List,
+    Dict,
+    Optional,
+    Any,
+)
+from pyspark.ml.torch.distributor import Distributor, TorchDistributor
+
+
+def write_to_location(location: str, content: str) -> None:
+    os.makedirs(os.path.dirname(location), exist_ok=True)
+    with open(location, "a") as f:
+        f.write(content)
```

Comment:
This does not need to be a util function. Just inline the implementation below

Reply:
depending on the SSHEnvManager code, we may use this a lot, hence why I made it a separate function. I just made it a util in the case we use this a lot.

Reply:
It still does not need to be in a util. This is a pretty common pattern in Python, and it's only 3 lines